apps/wallet/src/ui/app/components/transactions-card/index.tsx

```diff
@@ -66,7 +66,7 @@ export function TransactionCard({ txn, address }: TransactionCardProps) {
                 <CardImage type={ImageType.BgSolid} shape={ImageShape.SquareRounded}>
                     <TxnIcon
                         txnFailed={executionStatus !== 'success' || !!error}
-                        variant={getLabel(txn, address)}
+                        variant={getTransactionAction(txn, address)}
```

Comment:
error is thrown on this line because we should update  TxnIcon component with TimelockedStaked, TimelockedUnstaked variants simmilar to what you added in the dashboard